<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FEditFull</class>
 <widget class="QWidget" name="FEditFull">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование записи</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/1/table_edit.png</normaloff>:/res/1/table_edit.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Название сеанса:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_seans"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>№ канала:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_kan">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>№ КА:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_nka">
         <property name="minimum">
          <number>126</number>
         </property>
         <property name="maximum">
          <number>127</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Дата и время начала сеанса:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dt_dtn">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Дата и время конца сеанса:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dt_dtk">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_sbr">
       <property name="text">
        <string>Повторить обработку</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Название исходного файла:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_isxf"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Название архивного файла:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_arxf">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>424</width>
         <height>66</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>424</width>
         <height>66</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnOk">
            <property name="text">
             <string>Записать</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/res/1/table_add.png</normaloff>:/res/1/table_add.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCancel">
            <property name="text">
             <string>Выход</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/res/Exit.png</normaloff>:/res/Exit.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnVost">
            <property name="text">
             <string>Восстановить</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/res/1/table_refresh.png</normaloff>:/res/1/table_refresh.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
